src/app/components/App/AppContainer.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import * as firebase from "firebase";
+import { connect } from 'react-redux';
+import { setAuthState } from '../../redux/actions.js';
+import App from './App.js';
+
+var config = {
```

Comment:
Gotcha. Well, at least for now the information is for a testing project. But if you could show me how to store it more safely, I'd be interested in learning!